icons/bootstrap/bootstrap-original.svg

```diff
@@ -0,0 +1 @@
MfNdmtJdXqKRhc0AEly8Ib7Yd4fX/jCFz8jBzAFKE/4HlZKYHdv738bsur0lI0O+BxAUsSF+1HiH/2XeHCCjuH1iS1tF7sOLN8YQDm/x1Y969evfX2kDSV+GAEkQcRTb5Fa8DJH9zh38Mu+w8tzzz13jJKp/NN7IzBkdgpdd9555/pAcvAJRFIijABqRZzHr49LWv+Sr+79LQA47U9Pv8M5gMmV+D0OstuefvrpftTT2akDGAEkTXVC/VrcEguK3/dl3w82f3Cl22//yfCLwiM5kAAcJ/vOyPdEgjACSIoR9PMjPrZRi7fhPJGnUCj+FACx+uXVJ48f136dHBTNlTignUHf++vXLM9t6u8dmN5BICXACCAhRiR8/xiAJ3onkBazt7IJfivQMzgW/P3/FADrpZUvHT979ok/HNyHwyrWgoz7/ty5c1/JbZm6AOwSlAkNQJXR34v1EXPJhnj94IAbkI/idE9YiiDChhj19KjPc3lRgLVx45bzJ7dOub2S4hcWMNB/YCXywtcH/ij8EcIuQC2I8fom8SslwzyxKSS3SliCIby32Np6VHlr6dKlbft7+h+Y2Dzlu4O9FfT8AFJN2PLjx370Pe80GIrQCIwARgBJUl4/X8cb7NNF6G0Dfg9dDlGfCtfTbADW/fffP/HSS+Zfn7GaPpntxRGqAgN+vgNZwKA88F8LFizYkUti/7/C0AAkQUywWsyntKVSnsf1XojxvH5QCJV+asAXbdx0003NX1vwtQvGN7dcoNzUbKcfhw5WSYapJrnxO9+761b4w39TV4CUCQ1ADSlG+AE8I+ANAFrIRwBejZHIcfiOQnt7u7jvvvsmHX/88Ue0tbZ/pKlxzBwB6xBI6xB3EIcO7qtu99HOiMHe/n1P3XLLLTtzSUHvr/8lZUIDUANiw31D/tixY+ft7xk4CQCEEMoSouiLfzgyj37sUCoXlmWLBiiklBTNCiKjXLTJrMo4PcUerQIIQDQMvn78zGPvQN7bexONAnljQAMwQmgAEqYMrw+lgGyPPR3A9PBCEUlFGRzh3/TWaiCxzFhsXvHrFX+zY8cOr7tD718laAASohzhD2epiEJxySUcdyRv61WK1BjVuW3npsUXX3zxeyj0/t46B/8qBA1AApTl9f3/FF23HOEXJNVI/HYD9u3t6/znI2ce+VPkRe99gUhvGZ8BqBA0ALUiVvhxhUKSShxfGA3CB4BUE/b09Hc/NO3Qqd9D/k6Hgl/87PtXGBqApKlwuF+0gEep8AEgNUZ1de3ddddh06fdh7zg9S8SAxR/VeCTgEkSoWuV/yekQMTuogJiQ95oEb8QQGacXLt+03sLNfF7Yb8e+vs+NVaTxh6k0AAkQZSuh/NK8/rKtxJx3LBm1FhKqTT6rebsC3fc+a3zjjtx9jOIFz+9fxVgF6BGjCjcr9N+PgBYNlRqjFy7q3vnv80+4uj7e3t7FfJ9/qD42e+vMowAqsOwf7Usq0B/Iw73Y48asqtahvs2kBorNw5YPQ9dduWlnzxs+qH39/b2OogWv3fLj6F/lWAEUFlCL9Kq3tYz5I8W4dsNIou0s+5Af8/zt91861333nvvHuRH9/V+PcVfA2gAKoc+Wu09mx/Q/R9HuG+nhRQZuU3CWbej64MnPvvZ83/2+uuvDyIfzntPJ+vC91ob7PNT/FWEBqB6uAAs27ZUqPjj7EG5t/Xi6lYSAdgpAZFWuyDcDxyZ3bKru+t3Ny+68fFHH320D/5+vC58k9cP9vkp/ipDA1BZXPjfyVeWZZsv4pH086OSKiQZkRsdErlfIywBYale2OgTQh6QUvZI6exy4O7o7O7+w4pnfvHCNddcsx3RQtaF7xrSg8aAVBkagOowfGFv3rzhjZZjx9+nbCmkVMKb2UdKNapn+5dKqsEBZ4+E62Szg519+3p2rN+0fvNrr73WuWjRoq5csaCAAbP31o2CE8hjyF9DRvVFWIdYyM+Lr8+Z502nBeQjBBsjn8EnKYKi1MXubZvKmKIBUz7FXyMYAVQW72LWp+gyzdbjhcH1aADCRBqMBsLu31P4o4h6uPjqDS8K8Cbn9AjOoV+v517F/I0Svbce7P+TGlGvF+FoxhTiB6fStg1po5W48N9URk/TvX0wndSY0X7x1StRRsBUdjRTrFCjIgJvm6IfZYz2i6+e0UVv8vj1eO7DxgL0dXr6OqIeL8J6I+j97UBePWAScXBarmIGCskoo14uwIOB4Jd96pGovj4hhBBCCCGEEEIIIYQQQgghhBBCCCGEEEIIIYQQQgghhBBCCCGEEEIIIYQQQgghhBBCCCGEEEIIIYQQQgghhBBCCCGEEEIIIYQQQgghhBBCCCGEEEIIIYQQQgghhBBCCCGE1AP/D9zQkSwrq6iWAAAAAElFTkSuQmCC"/></defs><path style="stroke:none;fill-rule:nonzero;fill:url(#a)" d="M14.121 16.727C13.88 8.023 19.281 0 26.586 0h74.836c7.305 0 12.71 8.023 12.469 16.727-.235 8.363.07 19.191 2.242 28.02 2.176 8.855 5.851 14.452 11.867 15.175v8.156c-6.016.723-9.691 6.32-11.867 15.176-2.172 8.828-2.477 19.656-2.242 28.02.242 8.703-5.164 16.726-12.47 16.726H26.587c-7.305 0-12.707-8.023-12.465-16.727.23-8.363-.07-19.191-2.242-28.02C9.703 74.399 6.016 68.802 0 68.079v-8.156c6.016-.723 9.703-6.32 11.879-15.176 2.172-8.828 2.473-19.656 2.242-28.02zm0 0"/><use xlink:href="#b" transform="scale(.5 .62745)"/></svg>
```

Comment:
The `B` text still looks blurry. Could you please use pure SVG format here?

Reply:
What do you mean? I kinda suck at svg lol

Reply:
Try searching in the file for `data:image`. You'll see an attribute where SVG embeds an image into the document. This should be avoided since rasterized images (JPG, PNG) don't scale as nice as SVG. I also don't know how it will behave when put through Icomoon.

See if you can convert that attribute to an SVG element. You might have to use an SVG editor (try Inkscape) to achieve this.

Reply:
No idea if it will work but I'll try